spec.html

```diff
@@ -25000,7 +24999,7 @@
         <emu-alg>
           1. Let _x_ be ? thisNumberValue(*this* value).
           1. Let _f_ be ? ToInteger(_fractionDigits_). (If _fractionDigits_ is *undefined*, this step produces the value 0.)
-          1. If _f_ &lt; 0 or _f_ &gt; 20, throw a *RangeError* exception. However, an implementation is permitted to extend the behaviour of `toFixed` for values of _f_ less than 0 or greater than 20. In this case `toFixed` would not necessarily throw *RangeError* for such values.
+          1. If _f_ &lt; -20 or _f_ &gt; 100, throw a *RangeError* exception.
```

Comment:
Currently, throwing behavior has nothing to do with the number receiver. I don't see a reason to change that--it would make things more complex for no particular user benefit that I can see.